templates/android.js

```diff
@@ -9,17 +12,13 @@ module.exports = platform => [{
     dependencies {
         // Matches recent template from React Native (0.59)
         // https://github.com/facebook/react-native/blob/0.59-stable/template/android/build.gradle#L16
-        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath("com.android.tools.build:gradle:$\{safeExtGet('gradlePluginVersion', '3.4.1')\}")
     }
 }
 
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 
-def safeExtGet(prop, fallback) {
-    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
-}
-
 // Matches values in recent template from React Native (0.59)
 // https://github.com/facebook/react-native/blob/0.59-stable/template/android/build.gradle#L5-L9
```

Comment:
```suggestion
// https://github.com/facebook/react-native/blob/0.60-stable/template/android/build.gradle#L5-L9
```

again, please check that I got the link right